src/main/java/com/minecolonies/coremod/client/render/worldevent/WorldEventContext.java

```diff
@@ -20,27 +19,130 @@
  */
 public class WorldEventContext
 {
-    public static final WorldEventContext INSTANCE = new WorldEventContext();
+    private static WorldEventContext instance;
+
+    /**
+     * In chunks
+     */
+    private RenderLevelStageEvent stageEvent;
```

Comment:
imho boilerplate to make these private/getter
like dont care, but intentionally made public to reduce code, since this class should ever only be available during render world event